package.json

```diff
@@ -50,6 +50,7 @@
     "mongoose-long": "0.1.1",
     "mongodb-topology-manager": "1.0.11",
     "node-static": "0.7.7",
+    "nsp": "^2.8.1",
```

Comment:
Nit: we generally don't allow ranges in package.json, and definitely don't allow `^`. `~` is ok in this particular case.

Reply:
sorry - this was the default from `npm install`...

IMO `^` should be fine for a tool like this, especially since it's a `devDependency`, but it's your call! 😉